app/views/shared/check_answers/_housing_benefit.html.erb

```diff
@@ -33,6 +34,7 @@
       <% regular_transaction_amount, regular_transaction_frequency = regular_transaction_answer_by_type(
            legal_aid_application: @legal_aid_application,
            transaction_type: housing_benefit_type,
+           owner_type: "Applicant",
```

Comment:
Non-blocking: further up the page you have `individual` set but this is hard coded as Applicant.  Do we only record Housing Benefit to Applicants?

Reply:
The question is only asked once. Depending on whether there's a and partner, the question reads either:
- "Does your client have any dependants?"
- "Does your client or their partner have any dependants?"

This is what `individual` is determining

Because there's only ever one lot of housing benefit, I've set it to be owned by the Applicant by default. It's either the applicant's or joint.

Do you think I could make all this clearer with better variable names or something else?

Reply:
No need for clarity - just tricky to read in the review format :+1: